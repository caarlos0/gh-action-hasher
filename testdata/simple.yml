name: CI
on: [push]
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: docker://alpine:3.18
      - uses: ./local/action@v1
      - uses: ../local/action@v1
      - name: foo
        uses: /abs/local/action@v1
      - uses: https://example.com/action@v1
      - width:
          a: b
        uses: actions/setup-go@v5 # comment
      - uses: caarlos0/meta/.github/workflows/govulncheck.yml@v0.1.0 # comment lalal
      - uses: docker/setup-qemu-action@v2 # comment
      - uses: docker/setup-buildx-action@v2
