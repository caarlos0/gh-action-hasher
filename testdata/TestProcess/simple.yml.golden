name: CI
on: [push]
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
      - uses: docker://alpine:3.18
      - uses: ./local/action@v1
      - uses: ../local/action@v1
      - name: foo
        uses: /abs/local/action@v1
      - uses: https://example.com/action@v1
      - width:
          a: b
        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
      - uses: caarlos0/meta/.github/workflows/govulncheck.yml@395f87b2cc4fbb0a99db4decb1b3bbd16bc07cfc # main
      - uses: docker/setup-qemu-action@2b82ce82d56a2a04d2637cd93a637ae1b359c0a7 # v2.2.0
      - uses: docker/setup-buildx-action@885d1462b80bc1c1c7f0b00334ad271f09369c55 # v2.10.0
